color.d

```diff
@@ -1777,14 +1779,66 @@ struct Point {
 	Point opBinary(string op)(int rhs) @nogc {
 		return Point(mixin("x" ~ op ~ "rhs"), mixin("y" ~ op ~ "rhs"));
 	}
+
+	Size opCast(T = Size)() inout @nogc {
```

Comment:
actually i think you meant T : Size here, not =. i'll do a fixup pass

Reply:
yes… oopsie!